Bottom: 7ebbc02159ccc7254855b2446c0da9d1cd32d975
Top:    82c387fa821ea0898070c557846013ef39f36c18
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 00:47:27 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
